### PR TITLE
basic tests of forwarders

### PR DESCRIPTION
**Pull request by famartinrh**
this PR includes refactor of existing ConnectorsTest and adds a ForwardersTest with basic test cases

### CODE REVIEW

**Comment by famartinrh** [comment]
@enmasse-ci run tests testcase=io.enmasse.systemtest.isolated.bridging.**

File: systemtests/src/test/java/io/enmasse/systemtest/isolated/bridging/ForwardersTest.java [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+        AddressUtils.waitForForwardersReady(new TimeoutBudget(1, TimeUnit.MINUTES), forwarder);
+
+        UserCredentials localUser = new UserCredentials("test", "test");
+        resourcesManager.createOrUpdateUser(space, localUser);
+
+        int messagesBatch = 50;
+
+        //send to address with forwarder
+
+        AmqpClient localClient = getAmqpClientFactory().createQueueClient(space);
```
Comment (lulf) [inline systemtests/src/test/java/io/enmasse/systemtest/isolated/bridging/ForwardersTest.java]: I'm not sure this client will get cleaned up properly (applies to connector tests as well I think?)

File: systemtests/src/main/java/io/enmasse/systemtest/utils/AddressUtils.java [hunk 2]
```diff
@@ -196,6 +210,13 @@ public static void waitForBrokersDrained(TimeoutBudget budget, Address... destin
         TimeMeasuringSystem.stopOperation(operationID);
     }
 
+    public static void waitForForwardersReady(TimeoutBudget budget, Address... destinations) throws Exception {
+        String operationID = TimeMeasuringSystem.startOperation(SystemtestsOperation.ADDRESS_WAIT_FORWARDERS);
+        Thread.sleep(2000);
```
Comment (kornys) [inline systemtests/src/main/java/io/enmasse/systemtest/utils/AddressUtils.java]: I think you should measure address wait after 2s sleep

Comment (lulf) [reply systemtests/src/main/java/io/enmasse/systemtest/utils/AddressUtils.java]: Shouldn't this sleep be removed? I don't understand why its there.

File: systemtests/src/test/java/io/enmasse/systemtest/isolated/bridging/ForwardersTest.java (hunk 1 above)
Comment (famartinrh) [reply systemtests/src/test/java/io/enmasse/systemtest/isolated/bridging/ForwardersTest.java]: it's cleaned properly, the amqpClientFactory used is managed and closed properly

File: systemtests/src/main/java/io/enmasse/systemtest/utils/AddressUtils.java (hunk 2 above)
Comment (famartinrh) [reply systemtests/src/main/java/io/enmasse/systemtest/utils/AddressUtils.java]: yes, I just copied from other methods that also had, I'm going to remove it

**Pull request merged**